ql/termstructures/yield/ratehelpers.cpp

```diff
@@ -545,7 +548,7 @@ namespace QuantLib {
         //    i.e. it can dynamically change
         // 2. input discount curve Handle might be empty now but it could
         //    be assigned a curve later; use a RelinkableHandle here
-        swap_ = MakeVanillaSwap(tenor_, iborIndex_, 0.0, fwdStart_)
+        auto tmp = MakeVanillaSwap(tenor_, iborIndex_, 0.0, fwdStart_)
```

Comment:
This could have stayed the same, right?  Later you can call `withFloatingLegConvention` on `swap_` and avoid the copy in `swap_ = tmp`.

Reply:
Right, I though `swap_` was still a `MakeVanilla`.